next-cmake/CMakeLists.txt

```diff
@@ -119,6 +121,18 @@ if(HIPBLASLT_ENABLE_LLVM)
 endif()
 if(HIPBLASLT_ENABLE_DEVICE)
     find_package(Python3 REQUIRED COMPONENTS Interpreter)
+
+    if(HIPBLASLT_ENABLE_ASAN AND NOT WIN32)
+        execute_process(
+            COMMAND ${CMAKE_CXX_COMPILER} --print-file-name=libclang_rt.asan-x86_64.so
+            OUTPUT_VARIABLE ASAN_LIB_PATH
+            COMMAND_ECHO STDOUT
+        )
+        string(STRIP ${ASAN_LIB_PATH} ASAN_LIB_PATH)
+        # Disable a few asan options to get builds going but these should be addressed
+        set(asan_opts "LD_PRELOAD=${ASAN_LIB_PATH}" "ASAN_OPTIONS=detect_leaks=0,new_delete_type_mismatch=0")
```

Comment:
Can you file an issue on new delete type mismatch? These are often real bugs that can lead to crashes in certain circumstances.